mmv1/third_party/terraform/website/docs/d/secret_manager_secret_version.html.markdown

```diff
@@ -6,7 +6,7 @@ description: |-
 
 # google\_secret\_manager\_secret\_version
 
-Get a Secret Manager secret's version. For more information see the [official documentation](https://cloud.google.com/secret-manager/docs/) and [API](https://cloud.google.com/secret-manager/docs/reference/rest/v1/projects.secrets.versions).
+Get the value and metadata from a Secret Manager secret version. For more information see the [official documentation](https://cloud.google.com/secret-manager/docs/) and [API](https://cloud.google.com/secret-manager/docs/reference/rest/v1/projects.secrets.versions). If you don't need the metadata (i.e., if you want to use a more limited role to access the secret version only), see also the `google_secret_manager_secret_version_access` resource.
```

Comment:
not sure if there's a way to do internal links / references? For now, I just wrapped the resource names in backticks

Reply:
You probably cloud try [`google_secret_manager_secret_version_access`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/secret_manager_secret_version_access). Plus, I assume you mean `google_secret_manager_secret_version_access` datasource instead of resource

Reply:
You probably could try [`google_secret_manager_secret_version_access`](https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/secret_manager_secret_version_access). Plus, I assume you mean `google_secret_manager_secret_version_access` datasource instead of resource

Reply:
@shuyama1 thanks
So would you hard-link to the terraform registry (i.e., `https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/secret_manager_secret_version_access` in your example)? There's no way to do a relative link and / or have it generate / template out the top level link?

Reply:
looking at the rest of the docs, it does seem like that's how it's done.

Reply:
Also, the other examples aren't using backticks once they add links, so copying that pattern for now.

Reply:
yeah, I think that's what've been implemented in other docs as well. We changed internal link to hard-link at some point.